### PR TITLE
Add ability to subscribe to on-chain aave events via Notifications or Webhooks

### PR DESCRIPTION
**Pull request by dillchen**

### CODE REVIEW

File: client/scripts/helpers/chain_notification_types.ts [hunk 1]
```diff
@@ -175,3 +175,35 @@ export const KulupuChainNotificationTypes = {
...
+
+export const DydxChainNotificationTypes = {
+  Governance: [
+    'proposal-canceled',
+    'proposal-created',
+    'proposal-executed',
+    'proposal-queued',
+    'vote-emitted',
+  ],
+  Token: [
```
Comment (jnaviask) [inline client/scripts/helpers/chain_notification_types.ts:187]: These token ones aren't going to work given the way the contracts are designed.

File: client/scripts/views/pages/notifications.ts [hunk 2]
```diff
@@ -776,6 +788,7 @@ const NotificationsPage: m.Component<{}, {
               selectedCommunity.network === ChainNetwork.Kulupu && m(KulupuChainEventNotifications),
               selectedCommunity.network === ChainNetwork.Kusama && m(KusamaChainEventNotifications),
               selectedCommunity.network === ChainNetwork.Polkadot && m(PolkadotChainEventNotifications),
+              selectedCommunity.network === ChainNetwork.Aave && m(DydxChainEventNotifications),
```
Comment (jnaviask) [inline client/scripts/views/pages/notifications.ts:791]: is network supposed to be Aave or Dydx here? Confused.

**Review by jnaviask — COMMENTED** [review]
One comment before merging, otherwise looks fine -- similar to other PR about how we're treating ChainNetwork.

File: client/scripts/helpers/chain_notification_types.ts (hunk 1 above)
Comment (dillchen) [reply client/scripts/helpers/chain_notification_types.ts:187]: Can you say more?

File: client/scripts/views/pages/notifications.ts (hunk 2 above)
Comment (dillchen) [reply client/scripts/views/pages/notifications.ts:791]: Both dydx and aave use ChainNetwork.AAVE right now

**Pull request merged**